crates/networking/manager/src/validate.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::{Ok, anyhow};
+use ream_beacon_chain::beacon_chain::BeaconChain;
+use ream_consensus::{
+    constants::MAX_BLOBS_PER_BLOCK_ELECTRA, electra::beacon_block::SignedBeaconBlock,
+    misc::compute_start_slot_at_epoch,
+};
+use ream_storage::tables::{Field, Table};
+
+#[derive(Debug)]
+pub enum ValidationResult {
+    Accept,
+    Ignore,
+    Reject,
```

Comment:
can you make gossipsub.rs into a folder then move this file their, but then make a folder called validate, and make a file called beacon_block and put `validate_beacon_block` there